modules/fts/pages/fts-creating-indexes.adoc

```diff
@@ -62,9 +68,14 @@ To create Full Text Index for documents in scope and collections, specify the fo
 
 [#enabling-scope-collections]
 == Enabling scope/collection(s)
-With collections, indexing and searching data from a single source collection would be the most common and natural use case. It works almost similar to the existing bucket-based index creation. Except that the user has to specify the scope and collection details while creating the index definition.
+The Full Text Search indexing is enhanced to include scopes and collections in the index definition.
+
+Some of the important benefits that you can get in indexing with scopes and collections are:
 
-If the user is indexing the default scope and collection, then the index creation steps look exactly similar to that of the pre-collection days.
+* Increased indexing efficiency as the Data Service can provide documents from specific collections to the Index Service.
```

Comment:
This adds ambiguity within documentation - the term "Index Service" is associated with global secondary indexes. For FTS, you'll need to use "Search service" or "Full Text Search indexes".

Reply:
fixed